projects/Mallard/src/helpers/fetch.ts

```diff
@@ -189,6 +190,30 @@ const fetchWeather = <T>(
     )
 }
 
+export const fetchCacheClear = async (): Promise<boolean> => {
+    try {
+        const response = await fetch(defaultSettings.cacheClearUrl)
+        const cacheNumber = await response.json()
+        const cacheNumberStorage = await cacheClearCache.get()
+        if (cacheNumberStorage === null) {
+            // No data, so store it
+            await cacheClearCache.set(cacheNumber.cacheClear)
+            return true
+        }
+
+        if (cacheNumberStorage !== cacheNumber.cacheClear) {
+            // Deletes downloaded issues and the cache clear - login and GDPR settings need to be kept
+            await deleteIssueFiles()
+            await cacheClearCache.reset()
+            return false
+        }
+
+        return true
+    } catch (e) {
+        return false
```

Comment:
Is false a useful default for _all_ errors here? E.g., someone accidentally removes the cache file on S3 and we then lose automatic downloading on app open (given the current usage of this function)?

Reply:
Yep I agree with this! Explicitly we want to know if we are _not ok_. So anything else we should just carry on.